docs/getting-started/github-apps/github-apps-landing-page.rst

```diff
@@ -0,0 +1,27 @@
+======================================================
+Dockstore Github App
+======================================================
+
+GitHub apps are a GitHub feature used to improve the interaction between external applications and GitHub.
+Users can grant a GitHub app specific permissions on the repos and/or organizations of their choosing.
+
+The Dockstore Github App adds support for automatically syncing workflows and is also used to register Services.
```

Comment:
The behavior for services should be no different than it is for workflows; this sentence makes it sound like it is. I would say "...automatically syncing workflows and services"

Reply:
The Dockstore Github App adds support for registering and automatically syncing workflows and services.